### ISSUE TITLE
Identical (overlapping) edges of different parts/volumes are collapsed

### ISSUE THREAD

**Issue by gnbl** [solvespace/solvespace#1051]
solvespace-openmp.exe 3.0~e74e2024

Edges that are identical seem to be suppressed, so multiple parts in an assembly appear to have one face (under the mouse cursor):

![image](https://user-images.githubusercontent.com/1181708/120484725-a3d2de00-c3b3-11eb-9846-ab7f2fbc3152.png)

Is this behavior intended and why?
Personally, I'd like to see that edge.

**Comment by gnbl** [comment]
Indeed, if the length of either edge is different, it is displayed.
Maybe a remnant of "union" behavior for modelling?

**Comment by phkahler** [comment]
I have vague memory of causing this as a result of fixing something else that was worse. It's still a valid issue.

Somewhere in the edge classification and rendering code...

**Comment by gnbl** [comment]
In case it helps, I have attached a small test case with an assembly of a cube and a boolean union.
[tmp.zip](https://github.com/solvespace/solvespace/files/6589827/tmp.zip)

- This only affects the resulting solid model edges, not individual group edges/outlines.
- It does not affect the faces, i.e. the union merges faces as expected, while the adjacent face of another entity remains separate.
